mkosi.profiles/kiosk/mkosi.extra/usr/lib/systemd/system-preset/05-particleos-kiosk.preset

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+disable systemd-homed-firstboot.service
```

Comment:
You will want to set `systemd.firstboot=no` on the kernel command line instead so `systemd-firstboot.service` is also skipped.

Reply:
Thinking about this, I suspect we do want firstboot, as if the user didn't set a root password in the config they should get prompted for one. But we definitely do not want homed, as the homedir will be used for gnome-kiosk.